site/en/performance/model_optimization.md

```diff
@@ -0,0 +1,134 @@
+# Model optimization
+As more machine learning models are deployed to mobile devices,
+inference efficiency has become a critical issue. Where the computational demand
+for *training* grows with the amount of models trained on different
+architectures, the computational demand for *inference* grows in proportion to
+the amount of users. We introduce the <b> Tensorflow Model Optimization Toolkit </b> to minimize complexity of inference: model size, latency and power consumption.
```

Comment:
Tensorflow Model Optimization Toolkit ? Should this link somewhere? Is it a separate product or a feature built in to toco or something?